static/js/WordSalad.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+
+export const WordSalad = ({ numLines, salad, renderItem }) => {
+/**
+ * This component renders a collection of text items, styled to fit within a specified
```

Comment:
This is good documentation. Avoid explaining how the component works and instead explain how to use it. E.g. don't write "This is used in the CSS to control layout." because users of the component don't need to know this.

Also, remove the requirement that each item has a text property (see comment below on using `nowrap` in CSS)